sklearn/utils/extmath.py

```diff
@@ -1021,7 +1027,7 @@ def _incremental_mean_and_var(
         # correction term of the corrected 2 pass algorithm.
         # See "Algorithms for computing the sample variance: analysis
         # and recommendations", by Chan, Golub, and LeVeque.
-        new_unnormalized_variance -= correction**2 / new_sample_count
+        new_unnormalized_variance -= correction ** 2 / new_sample_count
```

Comment:
Let's not introduce cosmetic changes irrelevant to the PR, for the sake of preserving git-blame history. This also goes against our linting style (hence current CI linting error).

```suggestion
        new_unnormalized_variance -= correction**2 / new_sample_count
```

Reply:
Thanks! It was indeed automatically formatted by Black, but I had a different version than `22.3.0`.